app/views/api/v1/models/_user.json.jbuilder

```diff
@@ -5,6 +5,7 @@ json.name resource.name
 json.nickname resource.nickname
 json.email resource.email
 json.account_id resource.active_account_user&.account_id
+json.linked_account_id resource.linked_account_id
```

Comment:
Let's create a separate Jbuilder for an account create method than the current partial. so we don't have to introduce the `linked_account_id`